frontend/app/src/pages/Connected/Home/index.js

```diff
@@ -1,266 +1,179 @@
-import React from 'react'
-import './style.scss'
+import React from "react";
+import "./style.scss";
```

Comment:
Seems your linter is configured differently than it should.

Reply:
Do you have prettier installed?

.prettierrc on root of the project should be

```
{
  "trailingComma": "es5",
  "tabWidth": 2,
  "semi": false,
  "singleQuote": true
}

```

Reply:
Fixed